source/code/scxsystemlib/disk/statisticallogicaldiskenumeration.cpp

```diff
@@ -234,6 +235,50 @@ namespace SCXSystemLib
             SCXCoreLib::SCXHandle<StatisticalLogicalDiskInstance> disk = *iter;
             disk->Update();
 
+            /***
+            * Fix for Bug10019956:
+            *
+            * If the mounted device has zfs file system and device name does not have "/" then it is a zpool name.
+            * Hence consider free space on it for total free space calculation.
+            * Else ignore it as it is a zfs partition on rpool.
+            * Here we should note that in zpool each and every zfs partition's free space is same as the zpools free space.
+            *
+            * Ex:
```

Comment:
Delete lines 241-246. I wasn't asking for english translation of your 'if' statement! :)  But the /etc/mnttab and output of 'df' was perfect in understanding the issue.